README.md

```diff
@@ -61,41 +61,48 @@ Run `examples/pounce.py` to replicate.
 
 
 # Setup
-Before installing coreax, make sure JAX is installed. Be sure to install the preferred
-version of JAX for your system.
 
-Install [JAX](https://jax.readthedocs.io/en/latest/installation.html) noting that there
-are (currently) different setup paths for CPU and GPU use:
+Install Coreax from PyPI by adding `coreax` to your project dependencies or running
 ```shell
-$ python3 -m pip install jax
+pip install coreax
 ```
 
-Install Coreax:
+Coreax uses JAX. It installs the CPU version by default, but if you have a GPU or TPU,
+see the
+[JAX installation instructions](https://jax.readthedocs.io/en/latest/installation.html)
+for options available to take advantage of the power of your system. For example, if you
+have an NVIDIA GPU on Linux, add `jax[cuda12]` to your project dependencies or run
 ```shell
-$ python3 -m pip install coreax
+pip install jax[cuda12]
 ```
 
-Optionally, install additional dependencies required to run the examples:
-```shell
-$ python3 -m pip install coreax[test]
-```
+There are optional sets of additional dependencies:
+* `coreax[test]` is required to run the tests and examples;
+* `coreax[doc]` is for compiling the Sphinx documentation;
+* `coreax[dev]` includes all tools and packages a developer of Coreax might need.
+
+Note that the `test` and `dev` dependencies include `opencv-python-headless`, which is
+the headless version of OpenCV and is incompatible with other versions of OpenCV. If you
+wish to use an alternative version, remove `opencv-python-headless` and select an
+alternative from the
+[OpenCV documentation](https://pypi.org/project/opencv-python-headless/).
 
-Should the installation fail, try again using stable pinned package versions. Note that
-these versions may be rather outdated, although we endeavour to avoid versions with
-known vulnerabilities. To install Coreax:
+Should the installation of Coreax fail, try again using stable pinned package versions
+provided in `uv.lock`. First,
+[install UV](https://docs.astral.sh/uv/getting-started/installation/). Then, run
 ```shell
-$ python3 -m pip install --no-dependencies -r requirements.txt
+uv sync
```

Comment:
In order to use the pinned versions, an end user would first have to clone the repository (to get the `uv.lock`) and _then_ sync. Even then, that's not perfect as that installs it as a project, rather than as a dependency of the _user's_ project, which is presumably what they'd want. Instead, we should probably recommend the use of `uv export` to generate a requirements.txt file, which can then be installed using the user's choice of Python package manager - not 100% sure on what the cleanest way of doing that for a separate project is, though.

Reply:
(However, this particular change could also be considered out of scope for this PR - it might be worth splitting out some of the README update into a separate PR?)